app_classes/person.py

```diff
@@ -4,7 +4,8 @@ def __init__(self,name,role,wants_accomodation):
 		self.name = name
 		self.role = role
 		self.wants_accomodation = wants_accomodation
-		self.rooms_allocated = []
+		self.office_allocated = ''
+		self.living_space_allocated = ''
 
 #Class Staff inherits from model
```

Comment:
Consider separating the hash sign with a space after the comment i.e:
 `# Class staff inherits from model`